src/index.js

```diff
@@ -39,205 +41,219 @@ module.exports = {
     scope: makeRule('scope'),
     'tabindex-no-positive': makeRule('tabindex-no-positive'),
   },
-  configs: {
```

Comment:
This is unchanged, just moved to the top level as its reused for the flat configs. Easier to check the diff having "hide whitespace" toggled.